### PR TITLE
[Storage] Update download requester pays sample

### PR DESCRIPTION
**Pull request by frankyn**
This PR is to update the download sample for requester pays. It also fixes a object type mismatch in the commented section.

@jabubake if I missed something on updating another requester pays sample with the recent fixes please let me know and I'll resolve the issues.

### CODE REVIEW

**Comment by frankyn** [comment]
Found an issue in my PR. Fixing now.

**Comment by frankyn** [comment]
PTAL, thank you.

File: google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java [hunk 1]
```diff
@@ -66,6 +66,7 @@
 import java.net.URL;
 import java.nio.ByteBuffer;
 import java.nio.file.Path;
+import java.nio.file.Paths;
```
[the remark this answers is not in the document]
Comment (frankyn) [reply google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java]: It is used in the commented section of the sample.

**Comment by frankyn** [comment]
Signed it!

**Comment by frankyn** [comment]
Thanks for your help @jabubake. I have updated the sample. PTAL.

**Comment by jabubake** [comment]
LGTM : might want to hold on merge/ doc update till a `google-cloud-java` release is cut.

**Comment by frankyn** [comment]
Thanks Jisha! Do you know when the next release will be out?

**Comment by frankyn** [comment]
Thanks @vam-google! I want to verify that the library was released and it should now contain the latest changes made for this PR to be able to merge. Is this the current state?

**Comment by vam-google** [comment]
I think the release was completed 2 days ago. @neozwu please confirm. But in general I don't see any reasons to not push this change.

**Comment by jabubake** [comment]
You are good to go on merge. See /releases for this repository

**Comment by frankyn** [comment]
Thank you @vam-google and @jabubake!

**Pull request merged**